packages/web3-eth-contract/test/unit/contract.test.ts

```diff
@@ -198,7 +198,7 @@ describe('Contract', () => {
 			// });
 			const deployedContract = await contract
 				.deploy({
-					input: GreeterBytecode,
+					data: GreeterBytecode,
```

Comment:
```suggestion
					input: GreeterBytecode,
```

Since `input` is the correct term